browser-tests/browser/chrome.go

```diff
@@ -0,0 +1,27 @@
+package browser
+
+import (
+	"context"
+	"github.com/chromedp/chromedp"
+)
+
+func NewChrome(headless bool) (context.Context, context.CancelFunc) {
+	var execCtx context.Context
+	var execCtxCancel context.CancelFunc
+	if headless {
+		execCtx, execCtxCancel = context.WithCancel(context.Background())
+	} else {
+		execCtx, execCtxCancel = chromedp.NewExecAllocator(context.Background(), append(chromedp.DefaultExecAllocatorOptions[:], chromedp.Flag("headless", false))...)
+	}
+
+	ctx, cancel := chromedp.NewContext(
+		execCtx,
+		//chromedp.WithDebugf(log.Printf),
+	)
+	go func() {
```

Comment:
If the subContext is done, you have to cancel the parent? Shouldn't this context be the parent context for the others?